src/filters/negative_filter.c

```diff
@@ -1,76 +1,66 @@
 #include <stdio.h>
 
-int negative_filter(const char *inputFile, const char *outputFile)
+int negative_filter(const char *input_file, const char *outputfile)
 {
-    FILE *fileIn = fopen(inputFile, "rb");
-    FILE *fileOut = fopen(outputFile, "wb+");
+    FILE *file_in = fopen(input_file, "rb");
+    FILE *file_out = fopen(outputfile, "wb+");
 
-    if (fileIn == NULL || fileOut == NULL)
+    if (file_in == NULL || file_out == NULL)
     {
         printf("File does not exist.\n");
-        if (fileIn != NULL)
-            fclose(fileIn);
-        if (fileOut != NULL)
-            fclose(fileOut);
+        if (file_in != NULL)
+            fclose(file_in);
+        if (file_out != NULL)
+            fclose(file_out);
         return 1;
     }
-
-    unsigned char *imageData = NULL;
-    unsigned char *newImageData = NULL;
+    unsigned char *image_data = NULL;
+    unsigned char *new_image_data = NULL;
     unsigned char headerInfo[54];
     unsigned char colorTable[1024];
-
-    // Read image header
-    fread(headerInfo, sizeof(unsigned char), 54, fileIn);
+    // read image header
+    fread(headerInfo, sizeof(unsigned char), 54, file_in);
```

Comment:
headerInfo was not renamed to  header_info like the rest of the files. similarly could be with colorTable. 
Consider using change all the same variables in the project method of an action so that it remains consistent throughout the code.   right Click on a name you wish to change and " change all occurrences" or  "ctrl + f2 " or Rename Symbols, that in theory should rename all of them across the files, but since its "C " language its not supported without extension so sadly have to remember :)

Reply:
Changed it to be inline with other filter file variable naming.